lib/orbf/rules_engine/data/decision_table.rb

```diff
@@ -4,11 +4,13 @@
 module Orbf
   module RulesEngine
     class DecisionTable
-      attr_reader :rules
+      attr_reader :rules, :start_period, :end_period
 
       HEADER_SEPERATOR = ":"
 
-      def initialize(csv_string)
+      def initialize(csv_string, start_period:, end_period:)
```

Comment:
How are we not breaking things here? Changing the initializer yet, al the specs still pass. What magic is this?

Reply:
I don't know it's not because we are still at ruby 2.5.8 ?

Reply:
I adapted all the `DecisionTable.new(csv) `with `DecisionTable.new(csv, start_period: nil, end_period: nil)`

Reply:
https://github.com/BLSQ/orbf2/pull/242/files#diff-f200d6914dc6602b8154241b5521488a66fe2880abf1ba32c140241a9f31965fL154